ttyplot.c

```diff
@@ -258,9 +266,13 @@ int main(int argc, char *argv[]) {
     int show_ver;
     int show_usage;
 
-    plotchar=T_VLINE;
-    max_errchar='e';
-    min_errchar='v';
+    setlocale(LC_ALL, "");
+    if (MB_CUR_MAX > 1)            // if non-ASCII characters are supprted:
+        plotchar.chars[0]=0x2502;  // U+2502 box drawings light vertical
+    else
+        plotchar.chars[0]='|';     // U+007C vertical line
+    max_errchar.chars[0]='e';
+    min_errchar.chars[0]='v';
```

Comment:
@edgar-bonet I think we may need `memset(x, sizeof(x), 0)` for all three of these prior to setting `.chars[0]` to ensure that the other bytes are also initialized.

Reply:
`plotchar`, `max_errchar` and `min_errchar` are statically-allocated globals. They are zeroed-out by the run-time at program startup.

Reply:
@edgar-bonet are you sure that global variables *without* keyword `static` — our case here — have "static storage duration"? I found these two related pieces in https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1256.pdf , draft version of C99:

This:
![static_automatic_Screenshot_20231117_151545](https://github.com/tenox7/ttyplot/assets/1577132/e06be94f-1d06-4231-aa29-801d54ec2b63)

And then (in **6.7.8**(.10)):

![c99_again_Screenshot_20231117_151722](https://github.com/tenox7/ttyplot/assets/1577132/b02373ba-f971-421c-8930-8b17c77947f6)

Isn't 6.2.4.**4** our case? it would be indeterminate according to 6.7.8.10 above then, or at least the compiler would have a right to, correct?

Reply:
6.2.4.3 is the case because the linkage is per default external for file scope objects.